vmdk_plugin/drivers/vmdk/vmdkops/esx_vmdkcmd.go

```diff
@@ -93,22 +93,28 @@ func (vmdkCmd EsxVmdkCmd) Run(cmd string, name string, opts map[string]string) (
 	ans := (*C.be_answer)(C.malloc(C.sizeof_struct_be_answer))
 	defer C.free(unsafe.Pointer(ans))
 
+	var ret C.be_sock_status
 	for i := 0; i <= maxRetryCount; i++ {
-		_, err = C.Vmci_GetReply(C.int(EsxPort), cmdS, beS, ans)
-		if err != nil {
-			var errno syscall.Errno
-			errno = err.(syscall.Errno)
-			msg := fmt.Sprintf("'%s' failed: %v (errno=%d).", cmd, err, int(errno))
-
-			// Still below maximum number of retries, log and continue
-			if i < maxRetryCount {
-				log.Warnf(msg + " Retrying...")
-				time.Sleep(time.Second * 1)
-				continue
-			}
-
-			if errno == syscall.ECONNRESET || errno == syscall.ETIMEDOUT {
-				msg += " Cannot communicate with ESX, please refer to the FAQ https://github.com/vmware/docker-volume-vsphere/wiki#faq"
+		ret, err = C.Vmci_GetReply(C.int(EsxPort), cmdS, beS, ans)
+		if ret != 0 {
```

Comment:
it happens all the time on alpine ::-), it simply means the errno is not properly reset (by someone else) and we may see the stale one. I did see it when working with the managed plugin on Alpine distro. It is explicitly a non-event for us since return 0 means we are totally good